card/src/main/java/com/adyen/checkout/card/CardComponentProvider.kt

```diff
@@ -101,11 +109,97 @@ class CardComponentProvider(
                 cardDelegate = cardDelegate,
                 genericActionDelegate = genericActionDelegate,
                 actionHandlingComponent = DefaultActionHandlingComponent(genericActionDelegate, cardDelegate),
+                savedStateHandle = savedStateHandle,
             )
         }
         return ViewModelProvider(viewModelStoreOwner, factory)[key, CardComponent::class.java]
     }
 
+    @Suppress("LongParameterList", "LongMethod")
+    fun get(
```

Comment:
The original `get` method and this one share a lot of code, we should see if we can extract the similar things

Reply:
Addressed it in 5e47fd8ecde09fca462588bdb0169c71c19d49e1